N: 5
task_1_weight: 1000.
task_2_weight: 1.
numberOfObstacle : 1
obsSize: {s0: 0.25, s1: 0.05, s2: 0.05}
obsCentre: {s0: 1.2, s1: 0.1, s2: 0.1}

